default/AI/ShipDesignAI.py

```diff
@@ -633,11 +635,11 @@ def evaluate(self):
         # However, we also need to make sure, that the closer we are to requirements,
         # the better our rating is so the optimizing heuristic finds "the right way".
         if self.fuel < self.additional_specifications.minimum_fuel:
-            rating += -1000 * (self.additional_specifications.minimum_fuel - self.fuel)
+            rating += MISSING_REQUIREMENT_MULTIPLIER * (self.additional_specifications.minimum_fuel - self.fuel)
```

Comment:
I'm fine with some discretionary extra whitespace for readability, and with three of us making discretionary decisions I won't worry about trying to get 100% consistency, but lets aim for at least moderate consistency.  In my note above I pointed out three nearly consecutive lines with the same substantive pattern but different whitespace patterns, so please either make the third like the first two or the first two like the third.